examples/module1/lesson1/validate-it/validation/validator.ts

```diff
@@ -0,0 +1,47 @@
+import { ValidationMethod } from './validation-methods';
+
+enum ValidationMessage {
+  SUCCESS_VALIDATION = 'Valid',
+  FAIL_VALIDATION = 'Invalid',
+}
+
+export function validator(validationMethods: ValidationMethod[]) {
+  const validationInput = document.getElementById(
+    'input'
+  ) as HTMLInputElement | null;
+  const validationButton = document.getElementById('button');
+  const clearButton = document.getElementById('button2');
+  const validationResult = document.getElementById('result');
+
+  if (
+    !validationButton ||
+    !clearButton ||
+    !validationInput ||
+    !validationResult
+  ) {
+    throw new Error('Some html element not found');
+  }
```

Comment:
Proponowałbym to wynieść do osobnej funkcji, nawet w tym samym pliku - verifyDOMContent itd.